GUIDE.md

```diff
@@ -2,15 +2,24 @@
 
 ## Safety & Sanitization
 
-If query args/params are passed via `%$` or `%?` they will not appear in query but will be appended to the arguments slice (2nd resulting param of `Builder.Build()` method).
+Query arguments should be passed via `%$` or `%?`, this way they won't appear in
+the query string but instead will be appended to the arguments slice (2nd return
+value of the `Builder.Build()` method), thus preventing potential SQL injections.
 
-`ExampleQuery1` in [examples_test.go](https://github.com/cristalhq/builq/blob/main/example_test.go) explicitly shows that arguments are not a part of `query` string but are in `args` slice.
+Examples in [example_test.go](example_test.go) explicitly show that query
+arguments are not a part of the `query` string but returned separately as the
+`args` slice.
 
-Such usage prevents potential SQL-injection and/or any other harmful user inputs. However, `%s` should be used with a care and no user input should be passed through `%s`,
+The `%s` verb should be used with an extra care, no user input should be passed
+through it.
 
 ## String placeholder
 
-To write just a string there is the `%s` formatting verb. Works the same as in the `fmt` package.
+To write just a string there is the `%s` formatting verb. Works the same as in
+the `fmt` package.
+
+Please note that unlike `fmt`, `builq` currently does not support width and
```

Comment:
Better to drop `currently` 'cause we're not going to implement it.

`s/width/width, precision`

Reply:
Not sure about precision because it's about numbers and we support only strings

Reply:
Fair, I just noted all fmt feature. Let's leave than.